src/Parser.js

```diff
@@ -789,7 +789,7 @@ Parser.prototype.parseSizeGroup = function(optional) {
     let res;
     if (!optional && this.nextToken.text !== "{") {
         res = this.parseRegexGroup(
-            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}$/, "size");
+            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}\s*$/, "size");
```

Comment:
How will this `\s*` interact with newlines?  I suppose KaTeX doesn't support the double newline -> `\par` conversion, but if it did, I suppose this regex might cause trouble...